test/integration/examples_build.cc

```diff
@@ -25,7 +25,7 @@
 #include "ignition/gazebo/test_config.hh"
 
 // File copied from
-// https://github.com/ignitionrobotics/ign-gui/raw/master/test/integration/ExamplesBuild_TEST.cc
+// https://github.com/ignitionrobotics/ign-gui/raw/ign-gui3/test/integration/ExamplesBuild_TEST.cc
```

Comment:
Should this link be updated? `ign-gui3` is for Citadel.

Reply:
This is referring to the original place where the file was, so I think it's ok to point to an older version.